README.md

```diff
@@ -54,15 +54,21 @@ For example, you can create business models with different pricing tiers, e.g.:
   - Price 1: 20 USD per month
   - Price 2: 200 USD per year
 
+#### Generate test data with the Stripe CLI
+
+The [Stripe CLI](https://stripe.com/docs/stripe-cli#install) `fixtures` command executes a series of API requests defined in a JSON file. Top help speed up the setup, we have added a [fixtures file](fixtures/stripe-fixtures.json) to bootstrap test product and pricing data in your Stripe account. Simply run `stripe fixtures fixtures/stripe-fixtures.json`.
+
+**Important:** Be sure to start the webhook forwarding (see below) so that the products created by the fixtures command above are imported into your database.
```

Comment:
Shouldn't this come _after_ the webhook section then?

Reply:
Yes, this should be moved underneath the webhooks setup section

Reply:
Do you mean this entire section?